client/src/components/ItineraryList/InterchangeInfo.tsx

```diff
@@ -0,0 +1,22 @@
+import logo from '../../static/img/stay-seated.svg';
+import { Leg } from '../../gql/graphql.ts';
+
+/**
+ * Displays an icon if a leg has a stay-seated transfer from the previous one.
+ */
+export function InterchangeInfo({ leg }: { leg: Leg }) {
```

Comment:
Personally I would hava named this component "InterchangeIcon" and moved the conditional out to the parent component:

```
leg.interchangeFrom?.staySeated && <InterchangeIcon />
```

but I think this is also ok, especially if we expect more conditions to be added (?)

Reply:
Yes, the idea is that you also add information about guaranteed transfers.